examples/airline_demo/airline_demo/notebooks/Delays_by_Geography.ipynb

```diff
@@ -22,18 +22,16 @@
     "from dagster import ModeDefinition, ResourceDefinition\n",
     "from collections import namedtuple\n",
     "\n",
-    "url = 'postgresql://{username}:{password}@{hostname}:5432/{db_name}'.format(\n",
-    "    username='test', password='test', hostname='localhost', db_name='test'\n",
+    "url = \"postgresql://{username}:{password}@{hostname}:5432/{db_name}\".format(\n",
+    "    username=\"test\", password=\"test\", hostname=\"localhost\", db_name=\"test\"\n",
```

Comment:
these quote changes seem less than ideal, assuming the escapes show up in the notebook when viewed in a tool. 

Do they?

Reply:
Just checked, they don't.